src/js/helpers/aggregatedAmountsHelper.js

```diff
@@ -10,6 +10,9 @@ export const determineSpendingScenario = (amounts) => {
     const current = amounts._combinedCurrentAwardAmounts;
     const potential = amounts._combinedPotentialAwardAmounts;
 
+    if (obligated === 0 && current === 0 && potential === 0) {
```

Comment:
Could you add a test for this case to [aggregatedAmountsHelper-test.js](https://github.com/fedspendingtransparency/usaspending-website/blob/dev/tests/helpers/aggregatedAmountsHelper-test.js)?